src/front/app/components/globals/Input.js

```diff
@@ -0,0 +1,80 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+const classNames = require('classnames');
+
+const inputTypes = [
+  // Input Based
+  'text',
+  'email',
+  'password',
+  'number',
+  'date',
+  'datetime',
+  'datetime-local',
+  'search',
+  'tel',
+  'color',
+  'file',
+  // Select Based
+  'select',
+  // Text Area Based
+  'textarea',
+];
+
+class Input extends Component {
+  static propTypes = {
+    inputType: PropTypes.oneOf(inputTypes).isRequired,
+    name: PropTypes.string.isRequired,
+    id: PropTypes.string,
+    className: PropTypes.string,
+    children: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.node,
+      PropTypes.instanceOf(Component),
+    ]),
+    rowCount: PropTypes.number,
+  };
+
+  static defaultProps = {
+    inputType: 'text',
```

Comment:
don't call this input type, call it type, as input require that prop either way.